OptionSelector/AnimalDetailTableViewController.m

```diff
@@ -0,0 +1,91 @@
+//
+//  AnimalDetailTableViewController.m
+//  OptionSelector
+//
+//  Created by Lauren Caponong on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "AnimalDetailTableViewController.h"
+#import "CQCategory.h"
+
+@interface AnimalDetailTableViewController ()
+
+@property (nonatomic) NSMutableArray *array2;
+
+
+@end
+
+
+
+
+@implementation AnimalDetailTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    
+    CQCategory *dog = [[CQCategory alloc] init];
+    CQCategory *cat = [[CQCategory alloc] init];
+    CQCategory *bug = [[CQCategory alloc] init];
+    
+    dog.name = @"Dog";
+    cat.name = @"Cat";
+    bug.name = @"Bug";
+    
+    [self.array2 addObject:dog];
+    [self.array2 addObject:cat];
+    [self.array2 addObject:bug];
```

Comment:
Why does your `AnimalDetailTableViewController` need its own copies of each animal category? Won't it be receiving these objects from your root view controller?